ai-pep: align qconv benchmark to conv

vkuzo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #36674 ai-pep: match the qlinear benchmark to linear
* **#36673 ai-pep: align qconv benchmark to conv**

Summary:

Slight changes to the qconv benchmark to make it match the floating
point benchmark, so we can compare across the two better.

Test Plan:

```
cd benchmarks/operator_benchmark
python -m pt.qconv_test --tag_filter all
python -m pt.conv_test --tag_filter all
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21102563](https://our.internmc.facebook.com/intern/diff/D21102563)

vkuzo committed:
ai-pep: align qconv benchmark to conv

Summary:

Slight changes to the qconv benchmark to make it match the floating
point benchmark, so we can compare across the two better.

Test Plan:

```
cd benchmarks/operator_benchmark
python -m pt.qconv_test --tag_filter all
python -m pt.conv_test --tag_filter all
```

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

z-a-f commented on benchmarks/operator_benchmark/pt/qconv_test.py:
```diff
@@ -11,17 +11,20 @@
         [1, 512, 512, 56, 56, 32, 3, 2, 1],
     ],
-    attr_names=["N", "IC", "OC", "H", "W", "G", "kernel", "stride", "pad"],
+    attr_names=["N", "in_c", "out_c", "H", "W", "G", "kernel", "stride", "pad"],
```
You can run individual tests by name using `--test_name QConv2dBenchmark_N1_IC2_OC3_H4_W5_G6_kernel7_stride8_pad9` (just an example), where X, Y, Z are the argument names, while 1, 2, 3 are the argument values. If you add the underscore `_` in the argument names, the test names will be `QConv2dBenchmark_N1_in_c2_out_c3_H4_W5_G6_kernel7_stride8_pad9`. I don't think this helps the naming of the test binaries and potential parsing.

vkuzo replied:
I'm flexible on what are the exact names - it's more important that the attribute names of the floating point version match the names of the quantized version so it's easy to compare them.  Just let me know what you folks prefer, and we can make both of them be consistent.

mingzhe09088 replied:
Thank you for aligning the names. Either is fine with me.

z-a-f reviewed (changes requested):
Test naming. I will leave it to @mingzhe09088 for approval

vkuzo committed:
Update on "ai-pep: align qconv benchmark to conv"


Summary:

Slight changes to the qconv benchmark to make it match the floating
point benchmark, so we can compare across the two better.

Test Plan:

```
cd benchmarks/operator_benchmark
python -m pt.qconv_test --tag_filter all
python -m pt.conv_test --tag_filter all
```

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]
